package.json

```diff
@@ -10,12 +10,17 @@
     "@types/node": "^20.3.1",
     "@types/react": "^18.2.12",
     "@types/react-dom": "^18.2.5",
+    "lodash.findindex": "^4.6.0",
```

Comment:
Nice that you added just those sub modules and not the complete Lodash package. This helps to keep the bundle size small.